soap/soap_connector.bal

```diff
@@ -16,35 +16,27 @@
 
 import ballerina/http;
 
-documentation {
-    SOAP client connector.
-    F{{clientEP}} Http client endpoint
-}
+# SOAP client connector.
+# + clientEP - Http client endpoint
 public type SoapConnector object {
 
     public http:Client clientEP;
 
-    documentation {
-        Sends request and expects a response.
-        P{{path}} Resource path
-        P{{request}} Request to be sent
-        R{{}} If success, returns the response object, else returns `SoapError` object
-    }
+    # Sends request and expects a response.
```

Comment:
It is better if we can add `#` between the description and the parameters.

Reply:
Fixed.